en/docs/tutorials/streaming-api/create-and-publish-sse-api.md

```diff
@@ -0,0 +1,141 @@
+# Create and Publish a Server Sent Events API
+
+Follow the instructions in this tutorial to design and publish API with a SSE backend, and thereafter invoke it using a cURL command.
+
+This tutorial demonstrates a SSE API which is connected to a backend that observes your system's memory, and gives you the values in a continuous manner.
+
+<html>
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>For more information on SSE APIs, see <a href="{{base_path}}/design/create-api/create-streaming-api/create-a-sse-streaming-api">Create a SSE API</a>.</p>
+</div> 
+</html>
+
+### Step 1 - Design a SSE API
+
+1.  Sign in to the WSO2 API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
+
+2.  Go to **CREATE API** and Click **Design New Streaming API**.
+
+    [![Design New Streaming API]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The <b>CREATE</b> button will only appear for a user who has the <code>creator</code> role permission.</p>
```

Comment:
Similar to [1].
[1] https://github.com/wso2/docs-apim/pull/3284/files#r589208308